presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -531,10 +535,17 @@ public Builder setEnvironment(String environment)
             return this;
         }
 
+        public Builder setBaseDataDir(Optional<Path> baseDataDir)
+        {
+            requireNonNull(baseDataDir, "baseDataDir is null");
+            this.baseDataDir = baseDataDir;
```

Comment:
@yingsu00 This can be shorted to `this.baseDataDir = requireNonNull(baseDataDir, "baseDataDir is null");`

Reply:
Done. @mbasmanova